src/renderer/pages/Recording.tsx

```diff
@@ -33,6 +33,13 @@ const Recording: React.FC<RecordingProp> = ({ isRecording, setRecording, recordi
     setDisabled(false)
   }
 
+  React.useEffect(() => {
```

Comment:
So this one handles that if both devices disconnected the recording time is Reset? What happens if only one device was recording from the beginning and then it disconnects?

Reply:
Okay.. I think this is okay... because the other state is also disconnected and the `&&` is satisfied... ignore this ;)